flow-core/src/utils/logger/structured-logger.ts

```
@@ -143,43 +153,60 @@ export class StructuredLoggerImpl implements StructuredLogger {
+   */
+  private shouldLogToChannel(messageLevel: LogLevel, channelLevel: LogLevel): boolean {
+    const messageLevelValue = this.getLogLevelValue(messageLevel);
+    const channelLevelValue = this.getLogLevelValue(channelLevel);
+    return messageLevelValue >= channelLevelValue;
+  }
+
+  /**
+   * Get numeric value for log level comparison
+   */
+  private getLogLevelValue(level: LogLevel): number {
+    const levels = { debug: 0, info: 1, warn: 2, error: 3, critical: 4 };
```

ESLint

[error] Unquoted property 'debug' found.

(quote-props)